.gitignore

```diff
@@ -104,3 +104,4 @@ db.sqlite3
  # vim python config
 .ycm_extra_conf.py
 
+.vscode/settings.json
```

Comment:
also this doesnt seem to ignore the .vscode folder

Reply:
actually the .vscode is in develop, it was merged to develop, I don't have it in my branch